cockroachdb/values.yaml

```diff
@@ -480,14 +478,6 @@ tls:
       group: cert-manager.io
       kind: Issuer
       name: cockroachdb
-      # Duration of Client certificates in hours
-      clientCertDuration: 672h
-      # Expiry window of client certificates means a window before actual expiry in which client certs should be rotated.
-      clientCertExpiryWindow: 48h
-      # Duration of node certificates in hours
-      nodeCertDuration: 8760h
-      # Expiry window of node certificates means a window before actual expiry in which node certs should be rotated.
-      nodeCertExpiryWindow: 168h
```

Comment:
And these?

Reply:
I'm not sure 😅 these were generated based on the release script. Should they be left in?

Reply:
Let me double check. The script shouldn't be touching those...